src/features/events/components/EventsList/index.tsx

```diff
@@ -9,6 +9,12 @@ import { Nav, List, ListItem } from './styled'
 import { ViewType, FilterType } from './types'
 
 import { useEvents } from '../../hooks/useEvents'
+import type { Event } from '../../types'
+
+const sorts = {
+  ascending: (a: Event, b: Event) => (a.startsAt < b.startsAt ? -1 : 1),
+  descending: (a: Event, b: Event) => (a.startsAt > b.startsAt ? -1 : 1),
+}
```

Comment:
Here we refactor the code to isolate the reused sorting functions. And look at that! They are pure – meaning giving the same input, they will _always_ return the same output. Very testable!